docs/microservices.json

```diff
@@ -27,6 +27,10 @@
         {
             "name": "CMC",
             "colour": "#f30000"
+        },
+        {
+            "name": "Evidence",
+            "colour": "#fff200"
```

Comment:
Hard to see this yellow on white background imho. Perhaps brown could be used instead? 
We're running out of colors 😨

Reply:
that's why i had 10 initial pastel colours init.

Reply:
will change the colour now

Reply:
![screenshot from 2018-01-10 11-50-06](https://user-images.githubusercontent.com/3073565/34771415-7075e4e0-f5fc-11e7-90a9-d693ebbbaed3.png)
How about 'http://www.color-hex.com/color/8b4513' http://www.color-hex.com/color/8b4513